CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 5.2.0
+  - Added support for disambiguating timestamps in daylight-savings overlap periods
```

Comment:
```suggestion
  - Added support for disambiguating timestamps in daylight saving time (DST) overlap periods [#92](https://github.com/logstash-plugins/logstash-integration-jdbc/pull/92)
```

Reply:
The changelog entry gets picked up for release notes, and it's helpful to have a link to the PR.